src/App.test.js

```diff
@@ -1,8 +1,13 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('displays hello message when button is clicked', () => {
```

Comment:
Here is another comment, just want to encourage you to try testing first next time :)